packages/story-editor/src/components/panels/design/videoOptions/test/videoOptions.js

```diff
@@ -74,4 +91,24 @@ describe('Panels/VideoOptions', () => {
     const panel = screen.getByText('Video Settings');
     expect(panel).toBeInTheDocument();
   });
+
+  it('should render volume input', () => {
```

Comment:
A karma test would be great for this new change

Reply:
What is the point of this on a none finalized version of UX?

Reply:
It will most likely remain a slider, so the test should not really change much.